### PR TITLE
Add OMG History to mkdocs.yml

### PR DESCRIPTION
**Pull request by bact**
This add `omg-history.md` (to be added by PR #1053) to `mkdocs.yml`

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ validation:
 nav:
 - 'Copyright': index.md
 - 'Introduction': introduction.md
+- 'History with OMG': omg-history.md
```
Comment (kestewart) [inline mkdocs.yml:37]: This needs to be an Annex.

**Review by bobmartin3000 — APPROVED** [review]
LGTM

File: mkdocs.yml (hunk 1 above)
Comment (bact) [reply mkdocs.yml]: Done.

**Review by goneall — APPROVED** [review]
LGTM

**Pull request merged**